src/EditorFeatures/TestUtilities/Formatting/CoreFormatterTestsBase.cs

```diff
@@ -218,12 +225,16 @@ internal void AssertFormatWithTransformation(Workspace workspace, string expecte
             Assert.True(newRootNodeFromString.IsEquivalentTo(newRootNode));
         }
 
-        internal static void AssertFormat(Workspace workspace, string expected, OptionSet optionSet, IEnumerable<AbstractFormattingRule> rules, ITextBuffer clonedBuffer, SyntaxNode root, IEnumerable<TextSpan> spans)
+        internal void AssertFormat(Workspace workspace, string expected, OptionSet optionSet, IEnumerable<AbstractFormattingRule> rules, ITextBuffer clonedBuffer, SyntaxNode root, IEnumerable<TextSpan> spans)
         {
             var result = Formatter.GetFormattedTextChanges(root, spans, workspace, optionSet, rules, CancellationToken.None);
             var actual = ApplyResultAndGetFormattedText(clonedBuffer, result);
 
-            Assert.Equal(expected, actual);
+            if (actual != expected)
+            {
+                _output.WriteLine(actual);
```

Comment:
what't he value of this?

Reply:
This gives [easy access](https://xunit.net/docs/capturing-output.html) to the actual result in Test Explorer, in addition to the usual diff output with pluses and minuses.